tests/unit/test_network_connections.py

```diff
@@ -3,34 +3,25 @@
 # SPDX-License-Identifier: BSD-3-Clause
 
 import itertools
-import os
 import pprint as pprint_
 import socket
 import sys
 import unittest
 import copy
 
-TESTS_BASEDIR = os.path.dirname(os.path.abspath(__file__))
-sys.path.insert(1, os.path.join(TESTS_BASEDIR, "../..", "library"))
-sys.path.insert(1, os.path.join(TESTS_BASEDIR, "../..", "module_utils"))
-
 try:
     from unittest import mock
 except ImportError:  # py2
     import mock
 
-sys.modules["ansible"] = mock.Mock()
 sys.modules["ansible.module_utils.basic"] = mock.Mock()
-sys.modules["ansible.module_utils"] = mock.Mock()
-sys.modules["ansible.module_utils.network_lsr"] = __import__("network_lsr")
 
 # pylint: disable=import-error, wrong-import-position
-import network_lsr
-import network_connections as n
-
-from network_connections import SysUtil
-from network_connections import Util
 
+from network_connections import IfcfgUtil, NMUtil, SysUtil, Util
```

Comment:
This breaks the previous import ordering style. It would be nice to keep it in the old way as much as possible. https://pypi.org/project/reorder-python-imports/ describes it. The changes here would be to first have "import ...." lines, then "from ..." lines and one "from ... " line for each class.

Reply:
I used `reorder-python-imports` on each file.  Hope that is ok.

Please submit a PR at https://github.com/linux-system-roles/tox-lsr to add an isort or reorder-python-imports test.